### PR TITLE
Mntor-3138-1 - Simple paypal tests fix. Merged with e2e-slack-integration-and-CI-fixes

### PR DESCRIPTION
**Pull request by mozilloid**
# References:

Jira: -
Figma:

# Description
Fixed the paypal tests.

# Screenshot (if applicable)

Not applicable.

# How to test

# Checklist (Definition of Done)

- [ ] Localization strings (if needed) have been added.
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [ ] I've added or updated the relevant sections in readme and/or code comments
- [ ] I've added a unit test to test for potential regressions of this bug.
- [ ] If this PR implements a feature flag or experimentation, the Ship Behind Feature Flag status in Jira has been set
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [ ] All acceptance criteria are met.
- [ ] Jira ticket has been updated (if needed) to match changes made during the development process.
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

File: src/e2e/pages/purchasePage.ts [hunk 1]
```diff
@@ -82,8 +82,7 @@ export class PurchasePage {
     await this.paypalButton.click();
     const newPage = await pagePromise;
     await newPage.waitForLoadState();
-    await newPage.waitForURL(/.*paypal\.com.*\/checkout.*/);
-
+    await newPage.waitForURL(/.*paypal\.com.*checkout.*/);
```
Comment (pdehaan) [inline src/e2e/pages/purchasePage.ts:85]: Why this change, and do we still need a slash after the domain?

ie: `/paypal\.com\/.*checkout/`

Also, what is the URL that we need the `.*` in the middle?
I tried signing up for VPN (in Canada) using SubPlat+PayPal and it gave me something like this URL:
```
https://www.paypal.com/checkoutnow?sessionID=qwerty
```

Curious if the domain is very different in staging that we can't simplify the RegExp

File: src/e2e/specs/purchase.spec.ts [hunk 2]
```diff
@@ -72,18 +72,15 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breach Scan, Monitor Plus Purchase
     // fill out subscription payment
     await purchasePage.authorizationCheckbox.check();
     await purchasePage.fillOutStripeCardInfo();
-    await purchasePage.payNowButton.click();
+    await purchasePage.payNowButton.click({ force: true });
```
Comment (pdehaan) [inline src/e2e/specs/purchase.spec.ts:75]: Q: Why does this need `{ force: true }`?
https://www.bondaracademy.com/blog/do-not-force-playwright

File: src/e2e/pages/purchasePage.ts (hunk 1 above)
Comment (mozilloid) [reply src/e2e/pages/purchasePage.ts:85]: yeah for some reason the links are different on firefox vs chrome when testing. I believe for firefox, it has 'intent=checkout' instead of '/checkout' so this regex captures it.

File: src/e2e/specs/purchase.spec.ts (hunk 2 above)
Comment (mozilloid) [reply src/e2e/specs/purchase.spec.ts:75]: I believe this is from the code I merged from rafee's [pr](https://github.com/mozilla/blurts-server/pull/4640#pullrequestreview-2105519406)

Comment (rafeerahman) [reply src/e2e/specs/purchase.spec.ts]: I ran into a flaky test once where the button was clicked/selected but nothing happened, thought this might help 🤷
